src/frontend/src/lib/stores/address.store.ts

```diff
@@ -1,24 +1,18 @@
-import { initCertifiedSetterStore } from '$lib/stores/certified-setter.store';
-import type { StorageStoreData } from '$lib/stores/storage.store';
 import type { Address, BtcAddress, EthAddress } from '$lib/types/address';
 import type { CertifiedData } from '$lib/types/store';
 import { writable, type Readable } from 'svelte/store';
 
-export type AddressData<T extends Address = Address> = CertifiedData<T>;
+type AddressData<T extends Address> = CertifiedData<T>;
 
-export type OptionAddressData = StorageStoreData<AddressData>;
+export type OptionAddressData<T extends Address> = AddressData<T> | undefined | null;
```

Comment:
Can you provide this change in a separate PR?

Reply:
will do

Reply:
PR https://github.com/dfinity/oisy-wallet/pull/2213

Reply:
Did you?

Reply:
ah, I meant, I will do after this one

Reply:
Oki doki

Reply:
Done in PR #2220